build_system.sh

```diff
@@ -14,10 +14,8 @@ BUILD_DIR="$DIR/build"
 OUTPUT_DIR="$DIR/output"
 
 ROOTFS_DIR="$BUILD_DIR/agnos-rootfs"
-ROOTFS_IMAGE="$BUILD_DIR/system.img.raw"
+ROOTFS_IMAGE="$BUILD_DIR/system.img"
 ROOTFS_IMAGE_SIZE=10G
```

Comment:
once this is changed to 5G, needs a comment explaining why (link here) and explain that we have to until most devices have the updater fix to bump it back up

Reply:
since resize2fs enlarge is already in place, I’m planning to leave 10G here, and just use “resize2fs -M” (Shrink the filesystem to the minimum size.) at the end.

since flashing a raw 10G image vs the minimum or 5G one takes longer. I can benchmark the time needed.

the minimum image or the 5G one flashed without memory errors.

Reply:
I added the 5G back, otherwise we could silently ship an image that's too big. It's unlikely but we should be very careful with anything involving the updater.